### ISSUE TITLE
.dt() is not functioning properly

### ISSUE THREAD

**Issue by cvernon3** [BYU-PRISM/GEKKO#37]
My python script is reporting that my derivatives are causing attribute errors as in quotations 
"
  File "C:\Users\corya\Anaconda3\lib\site-packages\gekko\gk_operators.py", line 36, in __getattr__
    raise AttributeError(name)
AttributeError: dt
"
I did not have this issue prior to updating GEKKO, so I think it may be related. Thanks!

**Comment by loganbeal** [comment]
@cvernon3, earlier versions of GEKKO incorrectly allowed differentials of MVs. It was removed in the recent release. Is this what you encountered?

@APMonitor, can you confirm that differentials are not allowed on MVs? I have the following quote from you from May 24th: 
> Any variable that is expressed as a differential in one of the equations should be an Var(), SV(), or CV().

**Comment by APMonitor** [comment]
Yes, differentials are only allowed on variables that are calculated.
